tests/foreman/maintain/test_upgrade.py

```diff
@@ -118,6 +124,7 @@ def test_negative_pre_upgrade_tuning_profile_check(request, custom_host):
     :expectedresults: Pre-upgrade check fails.
     """
     profile = request.node.callspec.id
+    rhel_major = settings.server.version.rhel_version[0]
```

Comment:
AFAIK this isn't a list in dynconf config, try str(settings.server.version.rhel_version)[0]?

Reply:
This is not a good idea. Consider RHEL 10 ...

Reply:
```suggestion
    rhel_major = custom_host.os_version.major
```

Reply:
Thanks @ogajduse that's what I'm looking for!